accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -391,6 +394,13 @@ continuation.
 
 #### Late fields and variables
 
+A non-local variable declaration of the form `late final v;`, `late final T v;`,
```

Comment:
I have a hard time reading this because it's not progressing logically. First it groups four declaration forms, then it says something for two of them, then it mentions "the rest" (non-final declarations, a syntactically simpler group) afterwards. I'm never sure what the thing I have just read really covers.

How about:

> A non-local variable declaration implicitly induces a getter into the surrounding scope.
> It also induces an implicit setter in the following case:
> * The variable declaration is non-final.
> * The variable declaration is final, late and has no initializer expression.
>
> \rationale{The final late no-initializer variable is special in that it is the only 
> final variable which can be assigned to, even if it can only be assigned once.}.

Reply:
That's more readable, done!